bluepysnap/network.py

```diff
@@ -156,14 +156,26 @@ def get(self, group=None, properties=None):
         if unknown_props:
             raise BluepySnapError(f"Unknown properties required: {unknown_props}")
 
-        res = pd.DataFrame(index=ids.index, columns=properties)
-        for name, pop in self.items():
+        # Retrieve the dtypes of the selected properties. However:
+        # - the category dtype is not preserved
```

Comment:
what's the reason for not preserving the category? maybe that explanation can be added.

Reply:
I was thinking it was the effect of the concatenation but I cannot reproduce the issue.
Since it seems that the category type is preserved, I'll remove this line from the comment, and leave only the one about the int type.

Reply:
Amending the previous comment: the categorical dtype wasn’t preserved when the Series were converted to numpy arrays to build `pop_df`, but they are preserved with the current code.